### DISCUSSION TITLE
Allow banning specific gates

### DISCUSSION THREAD

**Discussion by vurvdev** [wiremod/wire#2725] [Suggestions]
Currently only `wire_banned_gate_categories` exists.

This isn't super helpful if you only want to ban say the Entity "Position" gate and allow stuff like Entity "SteamID".

**Comment by vurvdev** [comment]
Also, a UI would be nice.

**Comment by vurvdev** [comment]
This is assuming it's not easy / not possible to do this from prop protections, if it is, maybe it'd be better to make documentation on how to go about doing this in a wiki page

**Comment by thegrb93** [comment]
The PlayerSpawnSENT hook works for blocking specific wire entities if I remember right. I had a modified 'evolve' admin mod that would use this to block certain ones.

**Reply by thegrb93** [reply]
Although I think it also had to block the tools as well which used CanTool I think.

**Reply by Serverregler** [reply]
I also want to restrict wire on certain entities and block tools aswell, is that something you would be able to help me with, or share that admin mod ?
